### PR TITLE
added irishTitles classes to files 

### PR DESCRIPTION
**Pull request by cyocum**
These were added where foaf:title identified the person as a king.

### CODE REVIEW

File: LL/senchas_síl_ébir.ttl [hunk 1]
```diff
@@ -591,25 +591,25 @@
     owl:sameAs <http://example.com/LL/clanna_ébir_i_l-leith_chuind.ttl#Conrach>.
 
 <#EochoAnchend>
-    a foaf:Person;
+    a irishTitles:Rí;
```
Comment (Eystein-Thanisch) [inline LL/senchas_síl_ébir.ttl]: Just to check, does `a irishTitles:Rí` in any way link all the kings together and imply they were all kings of the same polity? Most of the "kings" in this section are kings of Ireland from Munster, but here the kings are of Leinster and Connacht.

File: LL/senchas_síl_ébir.ttl [hunk 2]
```diff
@@ -655,7 +655,7 @@
     rel:descendantOf <#Dare>.
 
 <#MacCon>
-    a foaf:Person;
+    a irishTitles:Rí;
```
Comment (Eystein-Thanisch) [inline LL/senchas_síl_ébir.ttl]: Looking at the text again, I am not so sure that Mac Con is being described as a king. Do you want to have a look yourself? LL, Book VI, Section 5, p.1374, first para.

File: LL/senchas_síl_ébir.ttl [hunk 3]
```diff
@@ -1088,7 +1088,7 @@
     rel:hasMet <#Patric>.
 
 <#Patric>
-    a foaf:Person;
+    a irishTitles:Noíb;
```
Comment (Eystein-Thanisch) [inline LL/senchas_síl_ébir.ttl]: He isn't actually called a saint in the text (this is possibly my mistake). I think the clause, "qui omnes sancti & sanctae sunt", refers to the young people who joined him. Should we still be giving him this title?

**Comment by cyocum** [comment]
No, I haven't put in all the permutations of the kingship.  However, it is permitted as they are in the class of "kings" which is what irishTitle:Rí means. The polity for which they are the king is interesting but separate information (we may want to tease the polity information out later).   If we are searching for kings, I would expect to find all kings not just kings of a particular polity.

File: LL/eoganachta_casil.ttl [hunk 4]
```diff
@@ -1821,6 +1822,7 @@
     owl:sameAs <#Sechnasaig>.
 
 <#Rectabrat-22262fbc>
+    a irishTitles:Rí ;
     owl:sameAs <#Rechtabrat>;
     foaf:title "rig"@sga, "king"@en;
     foaf:nick "ind rige"@sga, "the king"@en;
```
Comment (cyocum) [inline LL/eoganachta_casil.ttl]: I am fine with these as they are.

File: LL/senchas_síl_ébir.ttl (hunk 2 above)
Comment (cyocum) [inline LL/senchas_síl_ébir.ttl]: I had a quick look.  I think it is referring to Mac Con but I am willing to revise this.

File: LL/eoganachta_casil.ttl (hunk 4 above)
Comment (Eystein-Thanisch) [inline LL/eoganachta_casil.ttl]: Really? To rephrase, _ind ríge_ does not mean "the king" and _in gaiscid_ does not mean "the spear". If we include these translations, users may think that our understanding of medieval Gaelic is rather superficial.

Comment (cyocum) [inline LL/eoganachta_casil.ttl]: You just stated that you were happy with "ind rige" to mean that he was king so I am rather confused.  I will update Fiachra.

**Pull request merged**